tap_xactly/helpers.py

```diff
@@ -0,0 +1,19 @@
+from singer import metadata
+from tap_xactly.streams import STREAMS
+from tap_xactly.client import XactlyClient
+
+
+def get_stream_object(stream, state, config):
+    return STREAMS[stream.tap_stream_id](get_client(config), state)
+
+
+def get_stream_schema(stream) -> dict:
+    return stream.schema.to_dict()
+
+
+def get_stream_metadata(stream):
+    return metadata.to_map(stream.metadata)
```

Comment:
I don't think these methods really count as helper methods. Since all of these take the `stream` parameter, it would be better to just put them in `streams.py`. It would make sense for a stream instance to know everything about itself, including the metadata and schema, and might also cut down on some code in `sync.py`.